### PR TITLE
Add Hugging Face link to ML documentation

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
### Description
_Describe what this change achieves._

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _ml-commons-plugin/pretrained-models.md [hunk 1]
```diff
@@ -44,11 +44,9 @@ The following table provides a list of sentence transformer models and artifact
...
+| `sentence-transformers/all-MiniLM-L6-v2` | 384-dimensional dense vector space.  | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/torch_script/sentence-transformers_all-MiniLM-L6-v2-1.0.1-torch_script.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-distilroberta-v1/1.0.1/torch_script/config.json) | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/onnx/sentence-transformers_all-MiniLM-L6-v2-1.0.1-onnx.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-distilroberta-v1/1.0.1/onnx/config.json) |
```
Comment (ylwu-amzn) [inline _ml-commons-plugin/pretrained-models.md:47]: This line is for model `all-MiniLM-L6-v2` , torchscript and onnx model config file are wrong `.../sentence-transformers/all-distilroberta-v1/1.0.1/torch_script/config.json` ,

File: _ml-commons-plugin/pretrained-models.md [hunk 2]
```diff
@@ -44,11 +44,9 @@ The following table provides a list of sentence transformer models and artifact
...
+| `sentence-transformers/all-MiniLM-L12-v2` | 384-dimensional dense vector space. | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L12-v2/1.0.1/torch_script/sentence-transformers_all-MiniLM-L12-v2-1.0.1-torch_script.zip)(https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/torch_script/sentence-transformers_all-MiniLM-L6-v2-1.0.1-torch_script.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/torch_script/config.json) | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L12-v2/1.0.1/onnx/sentence-transformers_all-MiniLM-L12-v2-1.0.1-onnx.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/onnx/config.json) |
```
Comment (ylwu-amzn) [inline _ml-commons-plugin/pretrained-models.md:48]: This line is for model `all-MiniLM-L12-v2`, the model zip  is `.../sentence-transformers_all-MiniLM-L6-v2-1.0.1-torch_script.zip`

File: _ml-commons-plugin/pretrained-models.md [hunk 3]
```diff
@@ -44,11 +44,9 @@ The following table provides a list of sentence transformer models and artifact
...
+| `sentence-transformers/all-MiniLM-L6-v2` | 384-dimensional dense vector space.  | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/torch_script/sentence-transformers_all-MiniLM-L6-v2-1.0.1-torch_script.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-distilroberta-v1/1.0.1/torch_script/config.json) | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/onnx/sentence-transformers_all-MiniLM-L6-v2-1.0.1-onnx.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/torch_script/config.json) |
```
Comment (ylwu-amzn) [inline _ml-commons-plugin/pretrained-models.md:47]: This line is for `all-MiniLM-L6-v2`, but seem wrong config file `.../all-distilroberta-v1/1.0.1/torch_script/config.json`

File: _ml-commons-plugin/pretrained-models.md [hunk 4]
```diff
@@ -44,11 +44,9 @@ The following table provides a list of sentence transformer models and artifact
...
+| `sentence-transformers/all-MiniLM-L12-v2` | 384-dimensional dense vector space. | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L12-v2/1.0.1/torch_script/sentence-transformers_all-MiniLM-L12-v2-1.0.1-torch_script.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L6-v2/1.0.1/onnx/config.json) | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L12-v2/1.0.1/onnx/sentence-transformers_all-MiniLM-L12-v2-1.0.1-onnx.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-MiniLM-L12-v2/1.0.1/onnx/config.json) |
```
Comment (ylwu-amzn) [inline _ml-commons-plugin/pretrained-models.md:48]: This line is for `all-MiniLM-L12-v2`, but see wrong config file `.../sentence-transformers/all-MiniLM-L6-v2/1.0.1/onnx/config.json`

**Review by ylwu-amzn — APPROVED** [review]

**Review by cwillum — APPROVED** [review]

**Review by carolxob — APPROVED** [review]

**Pull request merged**